tests/kernels/test_attention.py

```diff
@@ -6,14 +6,14 @@
 from xformers import ops as xops
 from xformers.ops.fmha.attn_bias import BlockDiagonalCausalMask
 
-from vllm._C import ops
+from vllm._C import ops, cache_ops
 from vllm.utils import get_max_shared_memory_bytes
 
 FLOAT32_BYTES = torch.finfo(torch.float).bits // 8
 # This will change depending on the compute capability.
 # - 512 as a buffer
 MAX_SEQ_LEN = get_max_shared_memory_bytes() // FLOAT32_BYTES - 512
-NUM_BLOCKS = 12000  # Arbitrary values for testing
+NUM_BLOCKS = 4321  # Arbitrary values for testing
```

Comment:
Why is this change needed?

Reply:
It will cause OOM even when fp8 is disable in test_paged_attention. The merged pr's ci's log have similar issue.